text/0015-expose-handeled-property.md

```diff
@@ -0,0 +1,43 @@
+* Start Date: 2022-09-26
+* RFC Type: feature
+* RFC PR: https://github.com/getsentry/rfcs/pull/15
+
+# Summary
+
+Allow a user to decide if an exception/error is considered `handled: true` or `handled: false`.
+
+# Motivation
+
+A common way to integrate Sentry into a PHP framework is to utilize the framework's error handler stack.
+
+```php
+use Psr\Http\Message\ServerRequestInterface;
+use Throwable;
+use function Sentry\captureException;
+
+public function logException(
+    Throwable $exception,
+    ?ServerRequestInterface $request = null,
+    bool $includeTrace = false
+): void {
+    captureException($exception);
+
+    parent::logException($exception, $request, $includeTrace);
+}
+``` 
+
+This creates the problem, that all exceptions/errors are now considered `handled: true`.
+
+# Options Considered
```

Comment:
An option to consider is actually exposing the whole [Sentry protocol](https://develop.sentry.dev/sdk/event-payloads/) and actually making it possible that you can create a SentryEvent, with a mechanism. either handled or not.

Reply:
In our framework integrations in the Python SDK we create a custom `_capture_exception()` methods that sets all the data that we want in the event and then call `capture_event`: https://github.com/getsentry/sentry-python/blob/master/sentry_sdk/integrations/flask.py#L204-L219